cmake/developer_package/ncc_naming_style/ncc_naming_style.cmake

```diff
@@ -114,21 +118,28 @@ endif()
 # SOURCE_DIRECTORY - directory to check sources from
 # ADDITIONAL_INCLUDE_DIRECTORIES - additional include directories used in checked headers
 # DEFINITIONS - additional definitions passed to preprocessor stage
+# CUSTOM_STYLE_FILE - custom style file path
```

Comment:
let's also add this option to a signature.

Reply:
I think it can be option like `STYLE_FILE` and if this option is empty we will use openvino default config

Reply:
do you mean to introduce cmake option called STYLE_FILE and remove CUSTOM_STYLE_FILE from args? 
something like that?
```
if(NOT DEFINED STYLE_FILE)
{
    set( STYLE_FILE "${ncc_style_dir}/openvino.style")
}
```

Reply:
Yes, I like the proposed fix

Reply:
I have added option to the singature